### PR TITLE
Links

### PR DESCRIPTION
**Pull request by GunnarPDX**
added functional links with proper routing, also made some changes to the nav/login form

### CODE REVIEW

**Comment by Celeo** [comment]
Which branch did you branch off? Lot of backend stuff in this PR.

File: frontend/src/components/Forms/SignupForm.jsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      setEmailError('Email is required');
+      return;
+    }
+
+    if (password.length <= 0) {
+      setPasswordError('Password is required');
+      return;
+    }
+
+    const response = await api.login({ username: email, password });
```
Comment (Celeo) [inline frontend/src/components/Forms/SignupForm.jsx:117]: If this is signing up for an account, it'll need to call to the [/users/](https://github.com/crowdsourcemedical/volunteer-database/blob/master/backend/app/routers/user.py#L44-L62) endpoint, not login.

File: frontend/src/components/Forms/SignupForm.jsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    }
+  };
+
+  return (
+    <Card className={classes.root} elevation={4}>
+      <Grid container direction="row" justify="space-between" alignItems="center" className={classes.header}>
+        <Typography variant="h4" component="h2" className={classes.headerLogin}>
+          Login
+        </Typography>
+        <a href="localhost:3000" className={classes.signUpLink}>
```
Comment (Celeo) [inline frontend/src/components/Forms/SignupForm.jsx:137]: Should this be a router link to somewhere?

File: frontend/src/index.css [hunk 3]
```diff
@@ -12,3 +12,7 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
+
+a {
```
Comment (Celeo) [inline frontend/src/index.css:16]: This is potentially a pretty big style change, right?

**Comment by GunnarPDX** [comment]
> Which branch did you branch off? Lot of backend stuff in this PR.

Shoot, I'm glad you noticed that.  I thought I was working off the frontend/staging, I must have checked out the backend/staging by accident.  I'll close this one and move changes over to the proper branch.

**Pull request closed without merging**